ros/src/waypoint_updater/setup.py

```diff
@@ -0,0 +1,12 @@
+## ! DO NOT MANUALLY INVOKE THIS setup.py, USE CATKIN INSTEAD
```

Comment:
This piece is a requirement from catkin in order to run python unit tests. Basically so unittest can access package code